### ISSUE TITLE
instance system is broken in current dev branch

### ISSUE THREAD

**Issue by ramagottfried** [ramagottfried/symbolist#108]
the application-restructuring branch MVC restructuring seems to have broken the instance capabilities of the previous version. currently when you open the max objects all symbolist objects are running the same instance! 

previously each instance of symbolist was separate.

**Comment by ramagottfried** [comment]
I haven't studied the changes, but I noticed it because first the max help patch wouldn't open due to a missing initialization of the `MessageManager` which was a bug that I had fixed already for the latest JUCE update. I needed to fix it again to get the help patch to open, which I did by adding `    MessageManager::getInstance()` to the SymbolistHandler constructor.  This creates the MessageManager if it didn't exist yet. 

Previously it wasn't needed until we opened the window -- so I wonder if there is some JUCE code being initialized too early here.

also, why do we need: `SymbolistHandler* SymbolistHandler::INSTANCE = NULL;` as a global variable?  each symbolist class it an independent instance -- maybe you meant to put this in the class rather than in the global scope?
